src/app/Scenes/Artwork/Artwork.tsx

```diff
@@ -84,6 +86,21 @@ export const Artwork: React.FC<ArtworkProps> = ({
     context,
   } = artworkBelowTheFold || {}
 
+  const getInitialAuctionTimerState = () => {
+    if (isInAuction) {
+      return currentTimerState({
+        isPreview: isPreview || undefined,
+        isClosed: isClosed || undefined,
+        liveStartsAt: liveStartAt || undefined,
+      })
+    }
+
+    return undefined
+  }
```

Comment:
No need to return `undefined`. This is the default return value anyway.
```suggestion
    }
  }
```